lib/editor/editor.dart

```diff
@@ -385,10 +407,13 @@ class EditorState extends State<Editor> {
                   ),
                 ),
                 onChanged: (String event) {
-                  handleTextChange(file, event, 'IN');
+                  handleTextChange(event, RegionPosition.inner);
+                  if (file.hasRegion) {
+                    handleRegionCaching(file, event, RegionPosition.before);
```

Comment:
```suggestion
                    handleRegionCaching(file, event, RegionPosition.inner);
```

I believe this should be the inner region